src/Extensions/SwiftypeSiteTreeCrawlerExtension.php

```diff
@@ -2,283 +2,15 @@
 
 namespace Ichaber\SSSwiftype\Extensions;
 
-use Ichaber\SSSwiftype\Service\SwiftypeCrawler;
 use SilverStripe\CMS\Model\SiteTree;
-use SilverStripe\CMS\Model\SiteTreeExtension;
-use SilverStripe\Control\Director;
-use SilverStripe\Core\Config\Config;
-use SilverStripe\Versioned\Versioned;
 
 /**
  * @property SiteTree|$this $owner
  */
```

Comment:
Don't need this, as it's defined in `SwiftTypeCrawlerExtension`.
```suggestion
```

You're using `getOwner()` in this file though, and we know this extension is meant to be applied to `SiteTree`, so it might be nice to add an `@method SiteTree|$this getOwner()`.